AutoRest/AutoRest.Core/Logging/CodeGenerationException.cs

```diff
@@ -9,7 +9,9 @@ namespace Microsoft.Rest.Generator.Logging
     /// <summary>
     /// Code generation aggregate exception.
     /// </summary>
+#if !DNXCORE50
```

Comment:
Let's just remove the serializable attributes for exceptions and add exclusions for any code analysis issues that show up